backend/game_handler.py

```diff
@@ -64,8 +130,15 @@ def get_multiplicator(self) -> float:
             )
             return multiplicator
 
+    def cashout(self, ws):
+        if ws in self.bids:
+            gain = self.get_multiplicator() * self.bids[ws]
+            self.bids.pop(ws, None)
```

Comment:
```suggestion
            self.bids.pop(ws)
```
The None has no effect here